js/interactive-guides/iguide-multipane.js

```diff
@@ -136,7 +136,8 @@ var iguideMultipane = (function () {
         if (tabbedEditor.length > 0) {
             if (currentView === 'single') {
                 if (!tabbedEditor.hasClass('disableContainer')) {
-                    tabbedEditor.css('height', 'auto');
+                    // no longer need to set auto height
+                    //tabbedEditor.css('height', 'auto');
```

Comment:
I think we need to put a fixed height for single column view. The height from the multi column pane coming in may not be the 400px height. The height could be set smaller or bigger than 400px when in multi column view. In single column view, we want a fixed height of 400px.